commands/eval_ro.md

```diff
@@ -0,0 +1,14 @@
+This is a read-only variant of the `EVAL` command that isn't allowed to execute commands that modify data. Unlike `EVAL`, scripts executed with this command can always be killed and never affect the replication stream. Because it can only read data, this command can always be executed on a master or a replica.
```

Comment:
New formatting guidelines

```suggestion
This is a read-only variant of the `EVAL` command that isn't allowed to execute commands that modify data.

Unlike `EVAL`, scripts executed with this command can always be killed and never affect the replication stream.
Because it can only read data, this command can always be executed on a master or a replica.
```